Patterns/Swedish/Swedish-NumbersWithUnit.yaml

```diff
@@ -765,13 +767,17 @@ FractionalUnitNameToCodeMap: !dictionary
     Hào: HAO
     Ngwee: NGWEE
 CompoundUnitConnectorRegex: !simpleRegex
-    def: (?<spacer>and)
+    def: (?<spacer>och)
 CurrencyPrefixList: !dictionary
   types: [ string, string ]
   entries:
 # Currency Prefix Symbols/Terms
+    Swedish krona: sek
```

Comment:
You should not have to add the ISO codes to the prefix list. This should be automatically part of the process.
Perhaps the code here got a bit out of date compared to English?